### ISSUE TITLE
VS Solution with fixed signatures?

### ISSUE THREAD

**Issue by Source1pt** [T1GxR/Axion-CS2-RAGE-CHEAT#5]
Anyone got a STABLE version of this project? by stable I mean fixed signatures and offsets.

**Comment by Dufiw** [comment]
> Anyone got a STABLE version of this project? by stable I mean fixed signatures and offsets.

Good luck getting it from anyone.

I've tried getting it from someone to mess with since it's considered a paste, but they all say 'the source is private'

**Comment by T1GxR** [comment]
> > Anyone got a STABLE version of this project? by stable I mean fixed signatures and offsets.
> 
> Good luck getting it from anyone.
> 
> I've tried getting it from someone to mess with since it's considered a paste, but they all say 'the source is private'

Bro, its easy to build it and fix the errors, just update the signatures and build as publish (very important). It should work if everything is well done.

**Comment by Source1pt** [comment]
Yeah the signature part is the problem, I can fix the other problems. just not the signatures

**Comment by vxnpp** [comment]
> Yeah the signature part is the problem, I can fix the other problems. just not the signatures

could u give us the fixed source?

**Comment by pogrammerx** [comment]
> Yeah the signature part is the problem, I can fix the other problems. just not the signatures

Ever heard of IDA? So IDA is this great program that helps you reverse certain applications, including Counter-Strike 2, you can even use it to update your sigs! Isn't that just great? So next time before making a request like this, try updating it yourself.

**Comment by finalsecure** [comment]
how to use IDA for get sigs? Any tips?

**Comment by finalsecure** [comment]
> > Yeah the signature part is the problem, I can fix the other problems. just not the signatures
> 
> Ever heard of IDA? So IDA is this great program that helps you reverse certain applications, including Counter-Strike 2, you can even use it to update your sigs! Isn't that just great? So next time before making a request like this, try updating it yourself.

bro how? can u drop a tutorial pls

**Comment by pogrammerx** [comment]
> > > Yeah the signature part is the problem, I can fix the other problems. just not the signatures
> > 
> > 
> > Ever heard of IDA? So IDA is this great program that helps you reverse certain applications, including Counter-Strike 2, you can even use it to update your sigs! Isn't that just great? So next time before making a request like this, try updating it yourself.
> 
> bro how? can u drop a tutorial pls

maybe use fucking google to search for a tutorial instead of spamming a almost 1 year old GitHub issue

**Comment by SlivcUX** [comment]
Why are we still replying to this? It's so old and shitty.

An besides @pogrammerX you don't need to be here you already have a decent src from Simon.

**Comment by pogrammerx** [comment]
> Why are we still replying to this? It's so old and shitty.
> 
> An besides @pogrammerX you don't need to be here you already have a decent src from Simon.

Why I'm replying, maybe because I'm getting email notifications of him replying to me.

And who said I need to be here?

Try using your Brain next time.
